addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,173 @@
+@o-theme-kanban-gray: #fcfcfc;
+
 .o_kanban_view.o_theme_kanban {
+    background-color: @o-theme-kanban-gray;
+
     &.o_kanban_ungrouped {
         .o-justify-content(center);
 
         .o_kanban_record {
-            .o-flex(0, 0, auto);
-            width: 100%;
-            @media (min-width: @screen-sm-min) {
-                width: 45%;
+            .o-flex(0, 1, auto);
+            width: 250px;
+
+            &:first-child, &:nth-child(2) {
+                @media (max-width: @screen-xs-max) {
+                    margin-top: 10px;
+                }
```

Comment:
mmm, you probably can't see the difference because of my percentage margins rules... 
Btw it's not important, I'll remove it